lib/process-services/src/lib/task-list/components/task-form/task-form.component.spec.ts

```diff
@@ -495,4 +499,139 @@ describe('TaskFormComponent', () => {
+            await fixture.whenStable();
+
+            const claimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-claim-button"]'));
+            expect(claimButton.nativeElement.innerText).toBe('ADF_TASK_LIST.DETAILS.BUTTON.CLAIM');
+        });
+
+        it('should not display the claim/requeue button if the task is not claimable ', async() => {
+            getTaskDetailsSpy.and.returnValue(of(taskDetailsWithOutCandidateGroup));
+
+            component.taskId = 'mock-task-id';
+            fixture.detectChanges();
+
+            await fixture.whenStable();
+            const claimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-claim-button"]'));
+            const unclaimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-unclaim-button"]'));
+
+            expect(component.isTaskClaimable()).toBeFalsy();
+            expect(component.isTaskClaimedByCandidateMember()).toBeFalsy();
+            expect(unclaimButton).toBeNull();
+            expect(claimButton).toBeNull();
+        });
+
+        it('should display the claim button if the task is claimable', async() => {
+            getTaskDetailsSpy.and.returnValue(of(claimableTaskDetailsMock));
+
+            component.taskId = 'mock-task-id';
+            fixture.detectChanges();
+            await fixture.whenStable();
+
+            const claimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-claim-button"]'));
+
+            expect(component.isTaskClaimable()).toBeTruthy();
+            expect(claimButton.nativeElement.innerText).toBe('ADF_TASK_LIST.DETAILS.BUTTON.CLAIM');
+        });
+
+        it('should display the release button if task is claimed by the current logged-in user', async() => {
+            getBpmLoggedUserSpy.and.returnValue(of(claimedTaskDetailsMock.assignee));
+            getTaskDetailsSpy.and.returnValue(of(claimedTaskDetailsMock));
+
+            component.taskId = 'mock-task-id';
+            fixture.detectChanges();
+            await fixture.whenStable();
+
+            const unclaimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-unclaim-button"]'));
+
+            expect(component.isTaskClaimedByCandidateMember()).toBeTruthy();
+            expect(unclaimButton.nativeElement.innerText).toBe('ADF_TASK_LIST.DETAILS.BUTTON.UNCLAIM');
+        });
+
+        it('should not display the release button to logged in user if task is claimed by other candidate member', async() => {
+            getTaskDetailsSpy.and.returnValue(of(claimedByGroupMemberMock));
+
+            component.taskId = 'mock-task-id';
+            fixture.detectChanges();
+            await fixture.whenStable();
+
+            const unclaimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-unclaim-button"]'));
+
+            expect(component.isTaskClaimedByCandidateMember()).toBeFalsy();
```

Comment:
Can we use toBe(true) and toBe(false) in all the occurrences? https://vincenttunru.com/toBeTruthy-vs-toBe-true/

Reply:
Updated